netbox/ipam/api/views.py

```diff
@@ -86,6 +88,7 @@ class PrefixViewSet(CustomFieldModelViewSet):
     filterset_class = filters.PrefixFilterSet
 
     @action(detail=True, url_path='available-prefixes', methods=['get', 'post'])
+    @advisory_lock(ADVISORY_LOCK_KEYS['available-prefixes'])
     def available_prefixes(self, request, pk=None):
         """
         A convenience method for returning available child prefixes within a parent.
```

Comment:
Can you please add a sentence or two in here about why we are now using the advisory lock decorator. Just so future NetBox historians understand it is in an effort to avoid a race condition.

Reply:
Certainly! Done